test/Plugins.js

```diff
@@ -0,0 +1,16 @@
+import {compile, serialize, pluginMiddleware, prefix} from "../index.js"
+
+describe('prefixer plugin', () => {
+    const prefixPlugin = node => {
+        switch (node.type) {
+            case 'decl':
```

Comment:
i'm not really sure how to prefix correctly `@keyframes` with the current approach used in `prefix` - it would make more sense to me if we could "insert" an additional node, rather than just prepending the prefixed version as a string - the same kinda could be said about prefixing declarations, but it's easier to not do that there (and more performant I bet) so it seems like a reasonable tradeoff there. 

I haven't checked it - but looking at how things are implemented it seems that with the current way of prefixing stuff it only prefixes `@keyframes` itself, but cannot recursively~ prefix its contents.